.github/workflows/pr.yml

```diff
@@ -26,8 +26,6 @@ jobs:
 
       - name: Setup Gradle
         uses: gradle/actions/setup-gradle@v4
-        with:
-          gradle-version: wrapper
```

Comment:
This has been the default since the beginning, but for some reason I like to be explicit about it.

Reply:
20926fd149d070905696fa3e8e4aa061f0651f91